docs/source/usage.rst

```diff
@@ -44,15 +44,18 @@ HTML
 ~~~~
 * ``--to html``
 
-  - ``--template full`` (default)
+  HTML Export.  Note on backward compatibility:  Be aware that if you were using custom copies of the old 5.x
+  template files (i.e. ``--template``), you will now need to use ``--template-file path/to/old/file.tpl``
+  in order to use that file in compatibility mode as opposed to other options.
+
+  - ``--template lab`` (default)
 
     A full static HTML render of the notebook.
-    This looks very similar to the interactive view.
+    This looks very similar to the jupyter lab interactive view.
 
-  - ``--template basic``
+  - ``--template classic``
 
-    Simplified HTML, useful for embedding in webpages, blogs, etc.
-    This excludes HTML headers.
+    Simplified HTML, using the classic jupyter look and feel.
```

Comment:
I think it might be useful to also document the “embeddable” version, which would be `--template-file=classic/base.html.j2` (or `lab/`)

Reply:
`--template=classic` is preferred over `--template-file=classic/base.html.j2`. It's possible in a future release that when the backwards compatibility is removed that the second pattern might not work that way anymore (`--template=classic --template-file=notbase.html.j2` might be required in some distance release, or no support template-file at all). https://nbconvert.readthedocs.io/en/6.0.2/changelog.html?highlight=--template-file#id3 outlines how to use template-file to refer to a specific `.tpl` 5.x file.

I'm hesitant to promote it more elsewhere for other uses because some of these always-supported patterns made the 6.0 upgrade harder. But I could be convinced it's a good idea to document it more if people feel strongly about it.

Reply:
`--template=classic` is not a replacement for `basic`/`classic/base.html.j2`, because it produces a full 13k line (?!) HTML document with styles and JS. This is not usable for embedding, at least not without using a HTML parser to get rid of the 13k lines of `<head>`. `classic/base.html.j2` produces just the notebook code and expects styles to be added in elsewhere (and we do it that way in Nikola).

Reply:
Hmm you're correct. I wasn't as involved with the html template changes from 5.x to 6.x. It looks like adding `--template-file=base.html.j2` along with `--template=classic` gets you back to close to the original `--template=base` pattern without all the styles and JS. Can you try that and see if it matches expectations? I can discuss with the other maintainers how to better preserve that capability in a clearer format with the new system if so.

Reply:
It seems to work, both from the command line and code. It would be good to have an officially recommended way to do this in the docs.

Reply:
I agree. Thanks for raising the concern and documentation gap. I'll talk with the other maintainers and see if we want to introduce an easier way to run this combination before I document it, but I don't believe we'll remove the pattern that worked for you here.